bin/react-to-product-owners-yml-changes.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+from __future__ import annotations
+
+import os
+import sys
+from os.path import dirname, join, realpath
+
+import yaml
+
+PREFIX = "Product Area: "
+
+
+def main():
+    # Run from project root.
+    os.chdir(realpath(join(dirname(sys.argv[0]), "..")))
+
+    product_owners_yml = yaml.safe_load(open(sys.argv[1]))
```

Comment:
Bit confused here - what is this argument supposed to be? Or rather, I get that the it's the path to the product owners file, but what does that look like? Is it from another repo that we import as a dependency?

Reply:
Yep, it's from security-as-code. That repo is cloned during the `cascade-product-owners-yml-changes.yml`
https://github.com/getsentry/security-as-code/blob/main/.github/workflows/cascade-product-owners-yml-changes.yml

Reply:
The PR's you've been reviewing in security as a code are being read here